yellowbrick/contrib/__init__.py

```diff
@@ -5,6 +5,3 @@
 #
 #
 # ID: __init__.py [a60bc41] nathan.danielsen@gmail.com $
-
-
-from .scatter import ScatterViz, ScatterVisualizer, scatterviz
```

Comment:
This was causing a recursive import that I couldn't figure out, so I've removed it from the import path. I don't think it will break backward compatibility too much.